Sources/LanguageServerProtocol/SupportTypes/Language.swift

```diff
@@ -37,10 +37,71 @@ public struct Language: RawRepresentable, Codable, Equatable, Hashable {
   }
 }
 
-extension Language: CustomStringConvertible {
-  public var description: String {
+extension Language: CustomStringConvertible, CustomDebugStringConvertible {
+  public var debugDescription: String {
     return rawValue
   }
+
+  public var description: String {
+    switch rawValue {
+    case "abap": return "ABAP"
```

Comment:
Why not switch on `self` (e.g. `case .abap`)?

Reply:
Good question. Because I didn’t think about that. Changed it to switch on `self`.